src/Type/Definition/Type.php

```diff
@@ -33,8 +35,8 @@ abstract class Type implements JsonSerializable
     public const FLOAT   = 'Float';
     public const ID      = 'ID';
 
-    /** @var Type[] */
-    private static $standardTypes;
+    /** @var array<string,ScalarType|CustomScalarType>> */
```

Comment:
```suggestion
    /** @var array<string, ScalarType|CustomScalarType>> */
```

Reply:
Heh, I'm dumb--I just figured out that I can commit these suggestions automatically. Cool.

Reply:
Haha, that's why I do it that way, to make ur life easier! :D